### ISSUE TITLE
Ajax support

### ISSUE THREAD

**Issue by hkarakose** [rzo1/crawler4j#245]
Hello

Is crawler4j capable of loading Ajax requests (or executing scripts) after html file is fetched?

**Comment by rzo1** [comment]
No. There is an open Pull Request for that features that requires some testing: https://github.com/rzo1/crawler4j/pull/236
